Makefile

```diff
@@ -124,37 +108,37 @@ cmd/bin/subctl-%: $(shell find cmd/ -name "*.go") $(VENDOR_MODULES)
 	GOARCH=$${components[-1]}; \
 	export GOARCH GOOS; \
 	$(SCRIPTS_DIR)/compile.sh \
-		--ldflags "-X 'github.com/submariner-io/submariner-operator/pkg/version.Version=$(VERSION)' \
+		--ldflags "-X 'github.com/submariner-io/subctl/pkg/version.Version=$(VERSION)' \
 		       -X 'github.com/submariner-io/submariner-operator/api/submariner/v1alpha1.DefaultSubmarinerOperatorVersion=$${DEFAULT_IMAGE_VERSION#v}'" \
         --noupx $@ ./cmd $(BUILD_ARGS)
 
-ci: golangci-lint markdownlint unit build images
```

Comment:
Why are these removed?

Reply:
Becasue these targets were removed in the earlier commit https://github.com/submariner-io/subctl/pull/5/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L177.

Reply:
These targets still exist in shipyard - we just removed the additional dependent targets in this file. So they should remain on this line although I'm not sure what this target is used for.